scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,124 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
```

Comment:
My first thought was "organization owner", but AFAIK extension doesn't know about that. The next alternative might be "organization role", but every org user has that permission, so it might not have much sense. _Maybe_ we should create an "organization admin" role for this?